Makefile

```diff
@@ -45,6 +45,15 @@ AI_MODELS_REGISTRY ?= modelregistry.azurecr.io
 AI_MODELS_REGISTRY_SECRET ?= modelregistry
 SUPPORTED_MODELS_YAML_PATH ?= ~/runner/_work/kaito/kaito/presets/workspace/models/supported_models.yaml
 
+## AWS parameters
+CLUSTER_CONFIG_FILE ?= ./docs/aws/clusterconfig.yaml
+AWS_KARPENTER_VERSION ?=1.0.8
+CLUSTER_NAME ?= kaito-aws
```

Comment:
it looks like this variable is tied to aws. can we add the prefix to it?  `AWS_CLUSTER_NAME`
This name is common and may be overwrite somewhere in such a long Makefile.

Reply:
Updated